go/vt/wrangler/testlib/rpc_reparent_external_test.go

```diff
@@ -44,8 +44,6 @@ func TestTabletExternallyReparented(t *testing.T) {
 	ctx := context.Background()
 	ts := memorytopo.NewServer("cell1", "cell2")
 	wr := wrangler.New(logutil.NewConsoleLogger(), ts, tmclient.NewTabletManagerClient())
-	vp := NewVtctlPipe(t, ts)
```

Comment:
Is this because you're planning to make a new, separate test for the vtctl command?

Reply:
I plan to duplicate this test for the wrangler implementation of TER. In that we can test the vtctl command.